### PR TITLE
write Example section

### PR DESCRIPTION
**Pull request by Habbie**

### CODE REVIEW

**Comment by RobinGeuze** [comment]
Should we also added code to generate it from the cert rather than the server already listening. I assume it would be handy for key rotation to generate the DS/CDNSKEY first when you generate the cert, before you actually start using it.

File: draft-vandijk-dprive-ds-dot-signal-and-pin/draft-vandijk-dprive-ds-dot-signal-and-pin.md [hunk 1]
```diff
@@ -94,6 +94,48 @@ The pseudo DNSKEY type can be used in CDNSKEY and CDS, as defined in RFC7344, re
...
+MIICIjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKCAgEAxH2a6NxIcw5527b04kKy
+...
+71AWASNoX2GQh7eaQPDD9i8CAwEAAQ==
+-----END PUBLIC KEY-----
+```
+
+To turns this into a CDNSKEY:
+
+1. remove the header and footer
+2. remove all newlines
```
Comment (chantra) [inline draft-vandijk-dprive-ds-dot-signal-and-pin/draft-vandijk-dprive-ds-dot-signal-and-pin.md]: If you want a 1-liner that will extract the content:
```
openssl s_client -connect ns.example.com:853 </dev/null \
  | openssl x509 -noout -pubkey \
  | sed '1d;$d' \
  | tr -d '\n'
```

Comment (Habbie) [reply draft-vandijk-dprive-ds-dot-signal-and-pin/draft-vandijk-dprive-ds-dot-signal-and-pin.md]: something like that is in poc/README.me; I decided not to copy it here to make the procedure more explicit for the reader

Comment (Habbie) [reply draft-vandijk-dprive-ds-dot-signal-and-pin/draft-vandijk-dprive-ds-dot-signal-and-pin.md]: or perhaps we can do both, I'll add the code

**Review by RobinGeuze — APPROVED** [review]

**Pull request merged**